source/constructs/cdk.json

```diff
@@ -2,6 +2,7 @@
   "app": "npx ts-node bin/live-streaming.ts",
   "context": {
     "aws-cdk:enableDiffNoFail": "true",
-    "@aws-cdk/core:stackRelativeExports": "true"
+    "@aws-cdk/core:stackRelativeExports": "true",
+    "@aws-cdk/aws-s3:serverAccessLogsUseBucketPolicy": true
```

Comment:
are there any logging buckets in this solution that are not for logging cloudfront? Like access logging for s3 buckets?